docs/firebase.md

```diff
@@ -0,0 +1,49 @@
+---
+id: firebase
+title: Set up Firebase
+---
+
+Honeycomb comes with methods and configurations for easy firebase and firestore cloud storage set up. Please use this section to configure firebase hosting services and firestore cloud storage.
```

Comment:
make `firebase` a link to the google docs on firebase - same with firestore and hosting services.